### ISSUE TITLE
bxt-rs cannot be injected at HL2005 or NGHL (3248 build)

### ISSUE THREAD

**Issue by SmileyAG** [YaLTeR/bxt-rs#41]
I suspect this is due to the fact that since in that build `hw.dll` is encrypted, and in bxt-rs there is no `LoadAndDecryptHwDLL` shit as it is done for **BunnymodXT**.

**Comment by YaLTeR** [comment]
Yeah. I want to give this another look in bxt-rs to see if maybe I want to implement it slightly differently.
